Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsStore.cpp

```diff
@@ -793,6 +793,11 @@ HashSet<RegistrableDomain> ResourceLoadStatisticsStore::domainsExemptFromWebsite
     if (!m_standaloneApplicationDomain.isEmpty())
         result.add(m_standaloneApplicationDomain);
 
+    for (auto& site : m_persistedSites) {
+        if (!site.isEmpty())
+            result.add(site.domain());
```

Comment:
So we only need registrable domains? If so, why are we sending Site objects to the NetworkProcess instead of RegistrableDomains?

Reply:
Yes, I made UI process only sends domains to network process.